dataprofiler/tests/profilers/profiler_options/test_unstructured_text_profile_options.py

```diff
@@ -0,0 +1,138 @@
+from dataprofiler.profilers.profiler_options import TextProfilerOptions, \
+    BooleanOption
+from dataprofiler.tests.profilers.profiler_options.test_base_inspector_options \
+     import TestBaseInspectorOptions
+
+
+class TestTextProfilerOptions(TestBaseInspectorOptions):
+
+    option_class = TextProfilerOptions
+    keys = []
+
+    def test_init(self):
+        option = self.get_options()
+        self.assertTrue(option.is_enabled)
+        self.assertTrue(option.is_case_sensitive)
+        self.assertIsNone(option.stop_words)
+        self.assertTrue(option.words.is_enabled)
+        self.assertTrue(option.vocab.is_enabled)
+
+    def test_set_helper(self):
+        option = self.get_options()
+
+        # validate, variable path being passed
+        expected_error = ("type object 'test.is_case_sensitive' has no "
+                          "attribute 'is_enabled'")
+        with self.assertRaisesRegex(AttributeError, expected_error):
+            option._set_helper({'is_case_sensitive.is_enabled': True}, 'test')
+
+        expected_error = ("type object 'test.stop_words' has no "
+                          "attribute 'is_enabled'")
+        with self.assertRaisesRegex(AttributeError, expected_error):
+            option._set_helper({'stop_words.is_enabled': True}, 'test')
```

Comment:
I think we are missing the two tests for:
`vocab.is_enabled` and `option.words.is_enabled`

Reply:
added